rollup.config.mjs

```diff
@@ -17,6 +17,12 @@ export default [
         format: 'umd',
         sourcemap: true,
       },
+      {
+        file: process.env.BUILD_OUTPUT || './dist/fabric.es.js',
+        name: 'fabric',
+        format: 'es',
+        sourcemap: true,
+      },
```

Comment:
this change makes it work with the 3 setup i tested so far, but is all rollup based, i want to check webpack too and and the standard next.js and SWC setup.

I m not saying i think we should have 2 builds ( or 3 ) but right now i m collecting data on how to ship something that works out of the box and is not tied to a gigantic fabric monolith.